frontend/src/components/Chat/DirectMessageSelect.tsx

```diff
@@ -69,22 +78,20 @@ export default function DirectMessageSelect({
             </Tr>
           </Thead>
           <Tbody>
-            {playersWithChats.map(player => (
-              <Tr key={player.id}>
-                <Td role='cell'>
-                  {player.userName}#{player.id.slice(-4)}
-                  {renderNotification(player)}
-                </Td>
-                <Td role='cell'>
-                  <Button
-                    onClick={() =>
-                      handleChat(player.id, `${player.userName}#${player.id.slice(-4)}`)
-                    }>
-                    Continue Chat
-                  </Button>
-                </Td>
-              </Tr>
-            ))}
+            {playersWithChats.map(participant => {

```

Comment:
might also be worth memoizing this as well, though it's not totally necessary

Reply:
decided to not do the memoization of this as it depends on handleChat which would then need to be wrapped with useCallback